src/crimson/osd/shard_services.h

```diff
@@ -492,8 +493,8 @@ class ShardServices : public OSDMapService {
     return {get_reactor_utilization()};
   }
 
-  auto get_or_create_pg(spg_t pgid) {
-    return pg_to_shard_mapping.get_or_create_pg_mapping(pgid);
```

Comment:
Can't we use the existing `get_or_create_pg_mapping` but also pass `seastar::this_shard_id()` from PGSplitting? I'm not sure that I understand the difference.

Reply:
@Matan-B there is no get_pg_to_shard_mapping() in ShardServices and pg_to_shard_mapping is private. So I kept the create_split_pg_mapping function but changed it to use get_or_create_pg_mapping. Is that okay?

Reply:
Yes, I've missed that. Thanks!